### DISCUSSION TITLE
After every update I need to reload without cache

### DISCUSSION THREAD

**Discussion by D3SOX** [netalertx/NetAlertX#570] [Q&A]
Is this a problem with my configuration? I'm using Firefox with some extensions.
After the latest update I thought again that my installation was broken but I just had to a <kbd>CTRL</kbd>+<kbd>F5</kbd> and it was fixed

**Comment by jokob-sk** [comment]
Hi @D3SOX ,

Thanks for the question.

I know this can be annoying - would a reminder after an update in the UI help (e.g.:  Application restarted - please refresh the cache by hitting CTRL+F5 .... etc. if you face issues).

Otherwise, can you help me slowly collect instances where that happened and I will try to address them one by one? This issue is a bit harder to reproduce but I'd like to solve it step-by-step.

Thanks for the help,
j

**Reply by D3SOX** [reply]
I think you might need to append hashes to the build files or there's Cache-Control header

**Reply by jokob-sk** [reply]
do you have any suggestions on how to do it - haven't dealt with this kind of issue before - ideally the cache would refresh on container restart - thoughts?

**Comment by pbo10** [comment]
I'm glad you posted this so I avoided wasting time troubleshooting what had gone wrong in the update. Now that I know the answer I remember it happened last time I updated as well but it took me longer to figure out that time. 

It's annoying because the GUI works, there are just no devices shown at all. I run it in docker and only update maybe once every 6 months so it's hard to remember about this issue.

**Comment by jokob-sk** [comment]
Hey @D3SOX @pbo10 ,

I tried to fix this in the next release. If you can, please have a look at the `pi.alert_dev` docker image, in about 15 minutes (or after the last [action finishes](https://github.com/jokob-sk/Pi.Alert/actions)) from now. 

It would be great if you could test this on your end by switching to the above image and letting me know if the issue was resolved.

Thanks in advance,
j

**Reply by D3SOX** [reply]
I switched the image to the dev one, did a normal reload and it definitely loaded the new version this time as seen by the footer and sidebar
![image](https://github.com/jokob-sk/Pi.Alert/assets/24937357/06a1ec31-dc7f-485c-a538-26f04467a8ee)
![image](https://github.com/jokob-sk/Pi.Alert/assets/24937357/baa305c9-5b70-43e9-b400-de542589c31c)

Unrelated but I don't like that there are no longer any icons in the sidebar

**Reply by jokob-sk** [reply]
Hey @D3SOX ,

you probably managed to download a DEV image where the menu was broken. The latest image has this kind of menu:

![image](https://github.com/jokob-sk/Pi.Alert/assets/96159884/37273990-1574-4402-9532-d74716bd4e27)

**Reply by jokob-sk** [reply]
Also, thanks for checking! 🙏

**Comment by pbo10** [comment]
Hi @jokob-sk 

I tested the dev image and the devices did show up without having to CTRL refresh, however when I went back to the latest stable image they also showed up without issue this time as well so I don't know if that's expected with the change or if whatever is normally happening during normal updates just didn't happen this time.

Thanks

**Comment by D3SOX** [comment]
FYI Had to reload without cache after updating to v24.6.8

Otherwise it looked like this
![image](https://github.com/jokob-sk/NetAlertX/assets/24937357/b2b31008-632e-4b0d-98f1-3b1db8ddffd2)
